Content.Server/Strip/StrippableSystem.cs

```diff
@@ -274,7 +274,13 @@ bool Check()
                 return true;
             }
 
-            var doAfterArgs = new DoAfterEventArgs(user, component.StripDelay, CancellationToken.None, component.Owner)
+            if (!_inventorySystem.TryGetSlot(component.Owner, slot, out var slotDef))
+            {
+                Logger.Error("The slot didn't exist.");
```

Comment:
The errors should be more descriptive, so that its clear what's going wrong when just looking at error logs in the console.

I.e., something like
```
$"{ToPrettyString(user)} attempted to place an item ({ToPrettyString(held)}) in a non-existent inventory slot (slot}) on {ToPrettyString(component.Owner)}"
```

Reply:
It errored on held, I guess because it's in an earlier if? So I just left that part out